OctoPod/File UI/FilesTreeViewController.swift

```diff
@@ -152,7 +152,7 @@ class FilesTreeViewController: UIViewController, UITableViewDataSource, UITableV
         if let imageView = cell.viewWithTag(50) as? UIImageView {
             imageView.image = UIImage(named: file.isModel() ? "Model_48" : "GCode_48")
             if let thumbnailURL = file.thumbnail {
-                octoprintClient.getThumbnailImage(path: thumbnailURL) { (data: Data?, error: Error?, response: HTTPURLResponse) in
+                octoprintClient.getThumbnailImage(path: thumbnailURL) { (data: Data?, _: Error?, _: HTTPURLResponse) in
```

Comment:
As coding style I like to be explicit about parameters for easier maintenance and readability. Would you be ok reverting this change for both params? Thanks